packages/inter-protocol/src/reserve/assetReserve.js

```diff
@@ -30,6 +30,15 @@ const nonalphanumeric = /[^A-Za-z0-9]/g;
  *   has not yet been compensated.
  */
 
+/**
+ * @typedef {{
+ *   state: {},
+ *   facets: { AssetReservePublicFacet, AssetReserveCreatorFacet },
```

Comment:
`facets` describes an object. I think you want something like,
```suggestion
 *   facets: {
 {
    publicFacet: AssetReservePublicFacet,
    creatorFacet: AssetReserveCreatorFacet,
    shortfallReportingFacet: unknown,
    limitedCreatorFacet: AssetReserveLimitedCreatorFacet,
    governedApis: unknown,
  }
```
I put `unknown` for what I didn' tknow

Reply:
done